assembly/__tests__/runtime/main.ts

```diff
@@ -557,3 +557,10 @@ export function setValue(value: string): string {
 export function getValue(): string | null {
   return storage.get<string>("name");
 }
+
+export function getDefaultValue(y: string = "hello world", x: i32 = 42): i32 {
+  assert(x == 42, "x is not default value of 42");
+  assert(y == "hello world", "y is not default value of hello world");
+  logging.log(y);
```

Comment:
nit: this is redundant with the assert

Reply:
Lol true, but it was just to see that when the test runs it prints `hello world`.  I can remove it though.

Reply:
Removed it ;-)